config/routes.rb

```diff
@@ -5,10 +5,8 @@
   get '/p/:id', to: 'pages#view', :as => 'view_page'
 
   # Contact form routes
-  resources :contact_forms, :only => [:new, :create, :response]
-  post 'contact' => 'contact_forms#create'
+  resources :contact_forms, :only => [:new]
   get 'contact' => 'contact_forms#new'
-  get 'contact_response' => 'contact_forms#response'
```

Comment:
Yes. I was using that as the MF response while working on dev because the standard MF response wasn't working correctly. However on production the MF default reaponse seems to be okay. Wouldn't mind leaving it in as a backup if it is no bother.